test/test_browser.py

```diff
@@ -124,6 +124,26 @@ def metafunc(self, with_wasm64):
   return metafunc
 
 
+def also_with_wasm2js_wasm64(f):
```

Comment:
maybe add "or", so: `also_with_wasm2js_or_wasm64`? when I first read this I thought it was all 4 possibilities, including wasm2js and wasm64. The `or` would make that more obvious.